src/harness/sas.py

```diff
@@ -29,8 +29,10 @@ def GetTestingSas():
   cbsd_sas_version = config_parser.get('SasConfig', 'CbsdSasVersion')
   sas_sas_version = config_parser.get('SasConfig', 'SasSasVersion')
   sas_admin_id = config_parser.get('SasConfig', 'AdminId')
+  maximum_batch_size = config_parser.get('SasConfig', 'MaximumBatchSize')
   return SasImpl(cbsd_sas_rsa_base_url, cbsd_sas_ec_base_url, sas_sas_rsa_base_url,\
-    sas_sas_ec_base_url, cbsd_sas_version, sas_sas_version, sas_admin_id), SasAdminImpl(admin_api_base_url)  
+      sas_sas_ec_base_url, cbsd_sas_version, sas_sas_version, sas_admin_id, maximum_batch_size),
+    SasAdminImpl(admin_api_base_url)
```

Comment:
not sure if this is really an issue or just the way it's  displaying in here.  When I copied and try to run it I got an indentation problem on line 35